_posts/2022-10-09-decentralization-of-zk-rollups.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+author: Ibrahim Yusufali
+title: Decentralization of ZK Rollups
+excerpt: "Zero knowledge rollups were used in the past couple of years to solve the scalability constraints in L1 monolithic chains. A lot of new projects have emerged in the space since then, each of them ambitious and hoping to be the “ultimate” solution. However, as it is with every rapidly growing technology there are certain aspects that may be overlooked, in order to facilitate fast execution and entry to the market. We will be doing a deep dive into what we believe is one of those aspects: “decentralization”."
+---
+
+_Many thanks to Preston Evans, Shashank Agrawal, Ventali Tan, Daniel Lubarov, Gautam Botrel and James Stearn for feedback._
+
+## Outline
+
+1. Introduction
+2. Upgrade Keys
+3. Why Proof of Stake (PoS) Might Not Work
+4. Proof of Efficiency
+5. Proof Generation Outsourcing + MEV Auction
+6. Our Proposals
+7. Conclusion
+
+
+## Introduction
+
+Zero knowledge rollups were used in the past couple of years to solve the scalability constraints in L1 monolithic chains. A lot of new projects have emerged in the space since then, each of them ambitious and hoping to be the “ultimate” solution. However, as it is with every rapidly growing technology there are certain aspects that may be overlooked, in order to facilitate fast execution and entry to the market. We will be doing a deep dive into what we believe is one of those aspects: “decentralization”. 
```

Comment:
I think this part should give more history of rollups in general. At the beginning, they were not created to really solve the scalability problem. Rather, some people realized some transactions can be settled and bundled more efficiently off-chain, instead of paying the fees and waiting for confirmations on-chain. Loopring (trading specific initially), plasma, state channel are some good examples. Zero-knowledge roll-up is just a more efficient way to do it, when people realized other solutions have various problems, and a broader range of applications. The Layer 2 / rollup talks in devcon 2022 (mostly in "Flowers" room) gave a really good overview on this, and spent a fair bit of time discussing centralization points and risks

Reply:
That would be interesting to add. I will try to make it concise as well, so as to not add too much length. Thank you.